BlakeDonn/rancid-tomatillos
# 7b

# Rotten Tomatillos PR template
### What's this PR do?  
- Adds favorite capability for movies if a user is logged in
- User can favorite either preview card of moviePage
- User can toggle between favorite view and regular view

### Where should the reviewer start?  
- Start by checking the changes to App and then moving to previewCard and moviePage

### How should this be manually tested?  
- Open the webpage and try toggling favorites
-  Switch between favorite view
- Make sure SVG status persists through previewCard and moviePage

### Any background context you want to provide?  
- Tried to make the Tomato status on App and pass in to the moviePage and previewCard components with no luck

### What are the relevant tickets? 
closes #86
closes #87
closes #88
